### DISCUSSION TITLE
Problemi pw per entrare in Raspian (via SSH)

### DISCUSSION THREAD

**Discussion by braudoo** [indomus/forum#646] [HUB: Home Assistant]
Ciao a tutti, ho un RP4 con Raspian + HA Core a cui mi collego in modalità headless.
Durante la procedura di aggiornamento di HA Core, questo si è interrotto improvvisamente
di fatto buttandomi fuori.

Quando ho riprovato a collegarmi a Raspbian (tramite SSH), la pw “Raspberry”, per l’utente “pi” 
non era più valida.

E possibile ripristinarla senza dover collegare monitor e tastiera?

Per quanto riguarda la procedura di aggiornamento questo è comunque andata a buon fine,
e curiosamente il servizio di HA, che avevo precedentemente fermato, è stato riavviato senza
che lanciassi il comando.

Grazie.

**Comment by KC-inDomus** [comment]
la password di default è raspberry, che io sappia. controlla semplicemente di non stare sbagliando la password

**Reply by snakuzzo** [reply]
Prova, ma considera che la procedura di aggiornamento di HA Core non c'entra niente con la password dell'utente `pi` o con qualsiasi altra password di qualche altro utente. Se è successo qualcosa dipenderà sicuramente da altro

**Reply by braudoo** [reply]
> la password di default è raspberry, che io sappia. controlla semplicemente di non stare sbagliando la password

Confermo che scrivo `raspberry` clin minuscolo.



> Prova, ma considera che la procedura di aggiornamento di HA Core non c'entra niente con la password dell'utente `pi` o con qualsiasi altra password di qualche altro utente. Se è successo qualcosa dipenderà sicuramente da altro

Collegherò il Rasp per entrare direttamente a questo punto
